wasm/module/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"syscall/js"
+
+	"github.com/psanford/wormhole-william/wasm"
+)
+
+func main() {
+	// make things available in JS global scope (i.e. `window` in browsers).
+	js.Global().Set("newWormholeClient", js.FuncOf(wasm.NewClient))
```

Comment:
As mentioned in the description, this should be consolidated into a single object for simplicity on the JS side.